app/controllers/admin_only/dashboard_controller.rb

```diff
@@ -0,0 +1,74 @@
+module AdminOnly
+
+  class DashboardController < ApplicationController
+
+    before_action :authorize_dashboard
+    before_action :set_data_gatherer
+
+
+    def index
+    end
+
+
+    def show
+    end
+
+
+    def show_recent_activity
+      respond_to do |format|
+        format.js
+      end
+    end
+
+
+    # WIP to change the timeframe for "recent" data.  See commented out .row in the _recent_activity partial
+    def update_timeframe
+      # params['data_gatherer']['timeframe']
+      if request.xhr?
+
+        begin
+          @data_gatherer.timeframe = params['data_gatherer']['timeframe'].to_i
+          head :ok
+        rescue ArgumentError
+          head :error # return an error to the xhr call
+        end
+
+      end
+
+    end
```

Comment:
This is not being used.  If the intent is to possibly use it in the future we can retain - but the comment should be corrected (this is _not_ referenced in `_recent_activity.html.haml`).